src/lib/markbind/src/parser.js

```diff
@@ -27,6 +27,11 @@ const ATTRIB_CWF = 'cwf';
 
 const BOILERPLATE_FOLDER_NAME = '_markbind/boilerplates';
 
+const PREFIX_TO_REDUCE_NAME_CLASH = '$__MARKBIND__';
```

Comment:
This acts like a "global variable prefix", maybe that would be a better name? The fact that it is done to reduce name clash should be a comment.

Reply:
Done. Named it `IMPORTED_VARIABLE_PREFIX`